sql-statements/sql-statement-create-index.md

```diff
@@ -142,6 +142,17 @@ Expression indexes have the same syntax and limitations as in MySQL. They are im
 
 Currently, the optimizer can use the indexed expressions when the expressions are only in the `FIELD` clause, `WHERE` clause, and `ORDER BY` clause. The `GROUP BY` clause will be supported in future updates.
 
+## Invisible index
+
+Invisible index is a new feature introduced in MySQL 8.0 that sets an index to invisible so that the optimizer no longer uses this index.
```

Comment:
```suggestion
Invisible indexes are a new feature introduced in MySQL 8.0 that sets an index to invisible so that the optimizer no longer uses this index.
```